src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23732.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue23732: _IssuesUITest
+	{
+		public Issue23732(TestDevice testDevice) : base(testDevice)
+		{
+		}
+		public override string Issue => "TabBar content not displayed properly";
+
+		[Test]
+		[Category(UITestCategories.TabbedPage)]
+		public void TabbedPageTabContentUpdated()
```

Comment:
This test is failing on Android:

```
Assert.That(label.GetText(), Is.EqualTo("page4"))
String lengths are both 5. Strings differ at index 4.
Expected: "page4"
But was:  "page2"
```

Reply:
I have updated the test cases based on your feedback to avoid the failing. Could you please review and let me know if you have any further concerns?